config/webpack.config.js

```diff
@@ -33,7 +44,71 @@ const globalEntryFiles = {
   ],
 };
 
-const configGenerator = (buildOptions, apps) => {
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
+  const manifests = getAppManifests();
+  let manifestsToBuild = manifests;
+  if (entry) {
+    const entryNames = entry.split(',').map(name => name.trim());
+    manifestsToBuild = manifests.filter(manifest =>
+      entryNames.includes(manifest.entryName),
+    );
+  }
+
+  return getWebpackEntryPoints(manifestsToBuild);
```

Comment:
```suggestion
  if (!entry) {
    return getWebpackEntryPoints(manifests);
  }
  const entryNames = entry.split(',').map(name => name.trim());
  return getWebpackEntryPoints(manifests.filter(manifest =>
    entryNames.includes(manifest.entryName),
  ));
```
I think early returns can be easier to read.